### ISSUE TITLE
About - P1 -  As a colour-blind user, I’m also having trouble seeing things because the colour contrast doesn’t work with my assistive tools 

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#5892]
Category - Accessibility

- [ref doc](https://docs.google.com/document/d/1op2sQfY_l73vkwZL1W20FtZvTDhNuuu_89aM4bhDGVQ/edit#)

Success metric: Meeting W3C A11y standards for people with colorblindness
